### PR TITLE
Auto wrong ball kick

### PR DESCRIPTION
**Pull request by Leaf-life**
This is Caiden, not Leif

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Associated Issue(s)
Issue: We don't have a relevant issue

## Steps to Test
(In order to fully test, we need a robot with a color sensor)
1. Ran in the sim, changed color sensor values + break sensors to get into the class. Didn't crash.

## Changelog

We'd like to eject balls when they're not ours at times.

## What?

Changed some classes in the loaderLogic stateMachine so that we have the option of running the hopper backwards at times if we want to.

## Why?

At times, it might benefit our strategy to not control the other alliance's balls.

### CODE REVIEW

**Review by hell0w0rld2 — DISMISSED** [review]

File: robot.py [hunk 1]
```diff
@@ -82,6 +82,11 @@ def createObjects(self):
         self.map = RobotMap()
         self.xboxMap = XboxMap(XboxController(1), XboxController(0))
 
+        self.driverStation = DriverStation.getInstance()
+
+        self.allianceColor = self.driverStation.getAlliance()
+        print("ALLIANCE COLOR:   ",self.allianceColor)
```
Comment (ManiacMoustache) [inline robot.py:88]: Remove this print

**Review by ManiacMoustache — CHANGES_REQUESTED** [review]

**Review by ManiacMoustache — DISMISSED** [review]
Looks good, the most recent commit was the best I've ever seen

**Review by ManiacMoustache — APPROVED** [review]
Looks good, despite the github magic

**Review by hell0w0rld2 — APPROVED** [review]
👍

**Pull request merged**